zerolog/zerolog.go

```diff
@@ -0,0 +1,70 @@
+package zerolog
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/airbrake/gobrake/v5"
+	"github.com/buger/jsonparser"
+	"github.com/rs/zerolog"
+)
+
+type Writer struct {
+	Gobrake *gobrake.Notifier
+}
+
+// New creates a new Writer
+func New(notifier *gobrake.Notifier) *Writer {
+	return &Writer{Gobrake: notifier}
+}
+
+// Write
+func (w *Writer) Write(data []byte) (int, error) {
+	lvlStr, err := jsonparser.GetUnsafeString(data, zerolog.LevelFieldName)
+	if err != nil {
+		return 0, fmt.Errorf("error getting zerolog level: %w", err)
+	}
+
+	lvl, err := zerolog.ParseLevel(lvlStr)
+	if err != nil {
+		return 0, fmt.Errorf("error parsing zerolog level: %w", err)
+	}
+
+	if lvl < zerolog.ErrorLevel || lvl > zerolog.PanicLevel {
+		return len(data), nil
+	}
+
+	var userData interface{}
+	err = json.Unmarshal(data, &userData)
```

Comment:
TLDR/speaking aloud: use `buger/jsonparser` when you are only interested in a specific key/piece of json and use `encoding/json` if you want to get the whole object.

My instinct was to use the jsonparser to unmarshal here but it doesn't seem to have a function to do that. This is probably because [the why it's fast section](https://github.com/buger/jsonparser#what-makes-it-so-fast) mentions the library avoids parsing the full record and avoiding encoding/json.